### DISCUSSION TITLE
API for org-level Dependency Insights

### DISCUSSION THREAD

**Discussion by captn3m0** [community/community#9331] [Other Feature Feedback, Questions, & Ideas]
The `https://github.com/orgs/:org/insights/dependencies` page returns a lot of useful information that is accessible org-wide. However, the API doesn't have any corresponding endpoints (even under GraphQL) to access this information. The currently accessible fields are under repository, so you have to fetch the same information on a repo-by-repo basis.

A similar request for licensing information is here: https://github.community/t/get-dependency-license-information-from-insights-api/197568

**Comment by JustinGOSSES** [comment]
Repo-by-repo is too slow for enterprises with a large number of GitHub organizations or just a large number of repositories. Really need this at the enterprise level or organization level.

**Comment by dgiessing** [comment]
This would be quite helpful with attacks like shai hulud, so you could query the whole organization at once.
